src/data/list/basic.lean

```diff
@@ -1791,13 +1784,30 @@ by { induction as, { refl }, { simp! [*, apply_ite (map f)] } }
 lemma last_map (f : α → β) {l : list α} (hl : l ≠ []) :
   (l.map f).last (mt eq_nil_of_map_eq_nil hl) = f (l.last hl) :=
 begin
-  induction l with l_ih l_tl l_ih,
+  induction l with l_hd l_tl l_ih,
   { apply (hl rfl).elim },
   { cases l_tl,
     { simp },
     { simpa using l_ih } }
 end
 
+lemma map_eq_repeat_iff {α β} {l : list α} {f : α → β} {b : β} :
```

Comment:
These should already be in scope
```suggestion
lemma map_eq_repeat_iff {l : list α} {f : α → β} {b : β} :
```

Reply:
True -- thanks. I'll remove them and then merge when CI is happy.